data/core/macros/amla.cfg

```diff
@@ -12,19 +31,323 @@
         [effect]
             apply_to=hitpoints
             increase_total=3
-            heal_full=yes
-        [/effect]
-        [effect]
-            apply_to=max_experience
-            increase=20%
-        [/effect]
-        [effect]
-            apply_to=status
-            remove=poisoned
-        [/effect]
-        [effect]
-            apply_to=status
-            remove=slowed
         [/effect]
+        {AMLA_EFFECTS_HEAL_AND_RAISE_XP}
     [/advancement]
 #enddef
+
+#
+# This is a resource intended for use in select mainline campaigns, and should not be relied upon for UMC.
+# It may change in ways that aren't backwards-compatible with or that negatively affect the balance of UMC content that uses it.
+#
+# Include this resource in your [campaign] tag to replace the default +3hp AMLA with more powerful AMLA choices.
+# All max-level units gain a +8hp AMLA option, and units with melee/ranged attacks gain options for increased melee/ranged damage.
+# Units whose melee attacks all have 1 strike gain a +4 melee dmg option. Units whose melee attacks all have <=2 strikes gain a +2 melee dmg option. Otherwise, units gain a +1 melee dmg option.
+# The same pattern follows for ranged attacks.
+#
+# The optional `use_stronger_amlas_filter` variable can be used to modify which units this macro affects. `use_stronger_amlas_filter` defaults to `side=1`
+[resource]
+    id=use_stronger_amlas
+
+    #############################
+    # INITIALIZE AMLA FILTER
+    #############################
+    [event]
+        name=prestart
+        [filter_condition]
+            [variable]
+                name=use_stronger_amlas_filter
+                equals=$null
```

Comment:
have you tested that this works? afaik [variable] checsk for attributes,not for tags

Reply:
Thanks - I just tried this out, and you're right that it doesn't work.

From testing, `use_stronger_amlas_filter.length` seemed to work. Does that seem like a reasonable solution?
```
        [filter_condition]
            [variable]
                name=use_stronger_amlas_filter.length
                equals=0
            [/variable]
        [/filter_condition]```

Reply:
yes that should work